src/target/riscv/debug_defines.c

```diff
@@ -1,7 +1,5 @@
-/*
- * This file is auto-generated by running 'make debug_defines' in
- * https://github.com/riscv/riscv-debug-spec/ (40b9a05)
- */
+/* SPDX-License-Identifier: BSD-2-Clause OR CC-BY-4.0 */
```

Comment:
Please, fix the SPDX identifier:
```suggestion
// SPDX-License-Identifier: BSD-2-Clause OR CC-BY-4.0
```
See https://github.com/riscv/riscv-debug-spec/pull/968#issuecomment-1958206773